hollow/src/main/java/com/netflix/hollow/api/codegen/objects/HollowObjectJavaGenerator.java

```diff
@@ -105,6 +106,9 @@ public String generate() {
         classBuilder.append("\n");
 
         classBuilder.append("@SuppressWarnings(\"all\")\n");
+        if (!className.equals(schema.getName())) {
+            classBuilder.append("@" + HollowTypeName.class.getName() + "(name=\"" + schema.getName() + "\")\n");
```

Comment:
That's gonna use the fully qualified name.  Can you declare an import?

Reply:
Sure, this was the lazy way to go